specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2018-03-01-preview/machineLearningServices.json

```diff
@@ -707,6 +707,98 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.MachineLearningServices/locations/{location}/updateQuotas": {
```

Comment:
>updateQuotas [](start = 102, length = 12)

I believe this should be modelled as PUT  /quotas/{quotaName} instead.. 
and GET /quotas for collection call.. that is the pattern that all RPs used.